### PR TITLE
Issue #7476  fixed

### PR DESCRIPTION
**Pull request by shivamtiwariwebkul**
## Issue Reference
#7476

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@shivamtiwariwebkul 
This is **working fine**. please find the **screenshot** [Here](https://webkul.chatwhizz.com/share/screenshot/642ed73483d74a61f2394b7e)

**Review by Sanjaybhattwebkul — APPROVED** [review]

File: packages/Webkul/Admin/src/Resources/lang/en/app.php [hunk 1]
```diff
@@ -1254,6 +1254,11 @@
             'update-success'                 => 'Cart rule updated successfully',
             'delete-success'                 => 'Cart rule deleted successfully',
             'delete-failed'                  => 'Error encountered while deleting Cart rule.',
+            'coupons' => [
+                'create-success'             => 'Coupon created successfully',
+                'update-success'             => 'Coupon updated successfully',
+                'delete-success'             => 'Coupon deleted successfully',
+                ]
```
Comment (jitendra-webkul) [inline packages/Webkul/Admin/src/Resources/lang/en/app.php:1261]: Fixed indentation here

**Comment by devansh-webkul** [comment]
@Sanjaybhattwebkul and @amit-webkul,

Retest this.

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Comment by Amitk-webkul** [comment]
@shivamtiwariwebkul 
notification close button is not set. [Here](https://tinyurl.com/2b44p6r5)

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**